src/static_array.cr

```diff
@@ -123,6 +219,12 @@ struct StaticArray(T, N)
     pointerof(@buffer)
   end
 
+  # Transforms `self` into a string
```

Comment:
Should be "Appends a string representation of this static array to the given IO'